src/containers/ConnectorManagement.jsx

```diff
@@ -137,7 +138,7 @@ export default class ConnectorManagement extends Component {
                 synchronize={() => this.synchronize()}
                 deleteAccount={idx => this.deleteAccount(accounts[selectedAccount])}
                 cancel={() => this.gotoParent()}
-                onSubmit={values => this.updateAccount(selectedAccount, values)}
+                onSubmit={values => this.updateAccount(connector, selectedAccount, values)}
```

Comment:
Typo during rebase I think:
`onSubmit={values => this.updateAccount(connector, accounts[selectedAccount], values)}`